src/controllers/ArtsController.js

```diff
@@ -1,7 +1,11 @@
+// eslint-disable-next-line import/prefer-default-export
+import sequelize from 'sequelize';
 import models from '../db/models';
 import { Response, Slugify } from '../helpers/index';
 
-const { Art, Media, Category } = models;
+const {
+  Art, Media, Category, User, Comment
+} = models;
 
 let response;
```

Comment:
Why is response a global variable?

Reply:
So it can be re-usable by many methods that want to return a response

Reply:
Since you are reassigning it in every method, there's no point declaring it globally.

Reply:
Okay, but that means I have to be using const to be declaring response wherever I'm using it